packages/phone-number-privacy/signer/src/common/database/wrappers/request.ts

```diff
@@ -47,3 +47,27 @@ export async function insertRequest(
     await (trx != null ? sql.transacting(trx) : sql)
   })
 }
+
+export async function deleteRequestsOlderThan(
+  db: Knex,
+  date: Date,
+  logger: Logger,
+  trx?: Knex.Transaction
+): Promise<Number> {
```

Comment:
```suggestion
): Promise<number> {
```

Reply:
The linter is gonna complain about Promise<Number> feel free to ignore and we can clean this up later though nbd